### ISSUE TITLE
Suggestion: Update prompt text for adding description when uploading an image to better explain its importance

### ISSUE THREAD

**Issue by stefanbohacek** [mastodon/mastodon#27731]
### Pitch

A comparison of existing prompt (left) and my suggested update (right).

![A side by side comparison of an image upload form on Mastodon. In the original on the left, text below the image says "No description added". A proposed alternative on the right: "Please describe your image for visually impaired."](https://github.com/mastodon/mastodon/assets/3909390/dc868994-b3e4-4b7c-903a-f06d28ab805c)

The exact wording is not as important, the point I would like to make is that the current text "No description added" may not fully convey the importance of why one should add image description.

### Motivation

I wrote a report on image description prevalence based on preliminary results of the [AltTextHealthCheck](https://mastodon.social/@AltTextHealthCheck) bot that tracks the percentage of captioned images that are posted on mastodon.social and other instances mastodon.social is federated with.

Summary: Some instances have good results, but things are not looking good, and slowly getting worse.

https://stefanbohacek.com/blog/alttexthealthcheck-image-accessibility-report-1/

**Comment by zoltan-dulac** [comment]
I 100+ this.  I hope this is implemented.

**Comment by rakoo** [comment]
As suggested by @stefanbohacek I'd like to remind that alt-texts do help visually impaired people, they also help everyone. As they say themselves in https://alttexthalloffame.org/#why-alt-text-important: 

> "And mind you, captioning your images doesn’t only help blind people. It can help people understand the context for the image, explain a pop-culture meme, provide a placeholder for a missing image, or image loading over a slow internet connection."

I'm only putting this to think about the content of the message. Not that the current one is bad, but is it enough ?

**Comment by AndyCarolan** [comment]
100% agree!

**Comment by oneeyedman** [comment]
YAS! +1

**Comment by rachellawson** [comment]
I like this! The new text you give actually fits in with the style of other UI text on the screen as it describes an action rather than a status.

**Comment by RiversideBryan** [comment]
💯 agree! Fantastic suggestion !

**Comment by stefanbohacek** [comment]
Thanks everyone!

To help quantify the potential impact of this change, here's a poll a ran a little while ago.

https://stefanbohacek.online/@stefan/111363818962489179

Only 179 people voted, but the results are still insightful, with 5% choosing the option "Not important to describe images".

Full results for reference:

> "Also, I'm curious, why do folks (typically) skip captioning images they upload?"
> 
> - Not important to describe images: 5%
> - Too difficult to come up with description: 8%
> - Not enough time: 7%
> - Physical challenges: 2%
> - Missing features: 1%
> - Something else?: 3%
> - I caption my images: 74%
>

**Comment by stefanbohacek** [comment]
Thanks everyone!

To help quantify the potential impact of this change, here's a poll a ran a little while ago.

https://stefanbohacek.online/@stefan/111363818962489179

Only 179 people voted, but the results are still insightful, with 5% choosing the option "Not important to describe images".

Full results for reference:

> "Also, I'm curious, why do folks (typically) skip captioning images they upload?"
> 
> - Not important to describe images: 5%
> - Too difficult to come up with description: 8%
> - Not enough time: 7%
> - Physical challenges: 2%
> - Missing features: 1%
> - Something else?: 3%
> - I caption my images: 74%
>

**Comment by ViStefan** [comment]
I think it will be reasonable to make alt text ckickable, to enter editing mode. Now I have to tap editing button on the photo, scroll down to textfield, enter the description and then press“back” being a little afraid if my changes will be saved. 🙂

**Comment by stefanbohacek** [comment]
> * Too difficult to come up with description: 8%

To additionally address this issue, I propose adding including a [simple example](https://mastodon.social/@alttexthalloffame/112026971409044465) on how to approach writing alt text as described on https://www.perkins.org/resource/how-write-alt-text-and-image-descriptions-visually-impaired/:

> Writing alt text: Identify who, expression, description, color, interesting features.

**Comment by stefanbohacek** [comment]
Yes, I really like how [Phanpy](https://phanpy.social/) handles this, which is the source of the screenshot. The alt text really should be on the same level of importance as the content of the post itself.

**Comment by rkingett** [comment]
I think the same form is used for uploading audio/video too, and I always add a transcript/description to my audio and videos I upload to Mastodon, so what would you suggest to cover uploading audio/video?

**Comment by Schoeneh** [comment]
Are there any updates to this important suggestion?
(One I support 100%, I really like how phanpy handles this & would like to see it adopted by 'the' flagship UI)
